presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -65,11 +65,32 @@ Date and Time Functions
 
 .. function:: from_iso8601_timestamp(string) -> timestamp with time zone
 
-    Parses the ISO 8601 formatted ``string`` into a ``timestamp with time zone``.
+    Parses the ISO 8601 formatted date and time ``string``, optionally with
+    time and timezone, into a ``timestamp with time zone``. Missing
```

Comment:
"time zone" should be two words (other usages in this document are consistent in this regard)

Let's clarify missing information:

> The time defaults to zero, and the time zone defaults to the session time zone.

I couldn't incorporate the word "missing" without having it twice, but I think the above is fine since we say "optionally" in the previous sentence.

We should also update the `current_timestamp()` documentation to say "session time zone" instead of "current time zone" (since it's not clear what "current" means).

Reply:
Maybe we should merge what there is now and create another PR after the merge to detail what "current" means (server or session or mixture)